### ISSUE TITLE
Different response data and status codes

### ISSUE THREAD

**Issue by patricks** [envoy/Ambassador#40]
Hi,

is there a way to switch the response data and response code on every request?

I have ui test which should return a 401 status code in the first two requests and in the third request it should return a 200 response. Is there a way to get this working?

**Comment by fangpenlin** [comment]
@patricks yes, but not with Ambassador's build-in response classes. I think that's a legit requirement, will think about it for the future version. But in the mean time, you can use lower level Embassy SWSGI application

https://github.com/envoy/Embassy#example
https://github.com/envoy/Ambassador/blob/master/Ambassador/Responses/DataResponse.swift#L62-L83

You can see these example about how to return any desired header with `startResponse` function
